utils/cmut__utils__compile_resource.cmake

```diff
@@ -0,0 +1,139 @@
+
+
+include("${CMAKE_CURRENT_LIST_DIR}/../cmut_message.cmake")
+
+
+function(cmut__utils__get_variable_name VARIABLE_NAME INPUT_FILE)
```

Comment:
this function should be extract in cmut__utils__string_to_variable_name.cmake
`cmut__utils__value_to_variable_name(result value)`

Reply:
done